store/src/java/com/zimbra/qa/unittest/TestScheduledTaskManager.java

```diff
@@ -29,21 +35,23 @@
 import com.zimbra.cs.mailbox.ScheduledTask;
 import com.zimbra.cs.mailbox.ScheduledTaskManager;
 
-public class TestScheduledTaskManager
-extends TestCase {
-
+public class TestScheduledTaskManager {
     static final String TASK_NAME = "TestTask";
     private static final String USER_NAME = "user1";
+    private static Mailbox mbox;
 
-    public void setUp()
-    throws Exception {
-        cleanUp();
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
@tmclane Should this method call `tearDown()`?  That seems to be the pattern that most of these test updates are following.